code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -118,6 +118,9 @@
 					emote_see = list("shakes its head", "shivers")
 					desc = "It's a corgi."
 					set_light(0)
+					flags = 0
+					atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+					minbodytemp = 250

```

Comment:
It won't work for `atmos_requirements` because it's a list, and initial() doesn't work for anything that has to be `new`'d, which includes lists. You'd have to make a new var to store the default value in.

Initial should work fine for `minbodytemp`, though, since it's a simple number.